Description: Add parentheses around 'a ? b : c ? d : e' expressions.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/turba-4.2.29/lib/Driver.php
+++ b/turba-4.2.29/lib/Driver.php
@@ -2632,25 +2632,28 @@
             case 'homeCountry':
                 $message->homecountry = !empty($hash['homeCountryFree'])
                     ? $hash['homeCountryFree']
-                    : !empty($hash['homeCountry'])
+                    : ( !empty($hash['homeCountry'])
                         ? Horde_Nls::getCountryISO($hash['homeCountry'])
-                        : null;
+                        : null
+                      );
                 break;
 
             case 'otherCountry':
                 $message->othercountry = !empty($hash['otherCountryFree'])
                     ? $hash['otherCountryFree']
-                    : !empty($hash['otherCountry'])
+                    : ( !empty($hash['otherCountry'])
                         ? Horde_Nls::getCountryISO($hash['otherCountry'])
-                        : null;
+                        : null
+                      );
                 break;
 
             case 'workCountry':
                 $message->businesscountry = !empty($hash['workCountryFree'])
                     ? $hash['workCountryFree']
-                    : !empty($hash['workCountry'])
+                    : ( !empty($hash['workCountry'])
                         ? Horde_Nls::getCountryISO($hash['workCountry'])
-                        : null;
+                        : null
+                      );
                 break;
 
             case 'email':
